### DISCUSSION TITLE
[Feature Request]: Using dev-proxy behind a corporate proxy

### DISCUSSION THREAD

**Discussion by coinzz** [dotnet/dev-proxy#1219] [Q&A] [answered]
### Description
The dev-proxy registers as a system-proxy. Therefore our corporate-proxy is not known to the dev-proxy. Our corporate-proxy is essential to use the internet at all. This leads to not having any internet access while the dev-proxy is on. Therefore we mostly can't even reach the parts of our application we would like to test with the dev-proxy.

### Expected behaviour
Possibility to configure a corporate-proxy for the dev-proxy and in best case also with a custom authentication.

### Dev Proxy Version
0.21.0

**Comment by waldekmastykarz** [comment, marked as answer]
When starting Dev Proxy, you can use `--as-system-proxy false` to have it not register itself as a system proxy. Then, you'd need to specify in your app to route its traffic through Dev Proxy. Alternatively, you can use `asSystemProxy: false` in devproxyrc.json.

**Comment by coinzz** [comment]
Ah thank you, we tried that and that did the trick :)
But unfortunately this option is missing in the [schema](https://[raw.githubusercontent.com/microsoft/dev-proxy/main/schemas/v0.21.0/rc.schema.json](https://raw.githubusercontent.com/microsoft/dev-proxy/main/schemas/v0.21.0/rc.schema.json)) and therefore not being suggested with auto-completion. I guess we overread the property in `devproxy -h`, but at least it is listed there. 
Suggestion to maybe add it to the `Getting Started` section, since otherwise it is also pretty hidden in the Documentation :)

**Comment by waldekmastykarz** [comment]
> But unfortunately this option is missing in the [schema](https://%5Braw.githubusercontent.com/microsoft/dev-proxy/main/schemas/v0.21.0/rc.schema.json%5D(https://raw.githubusercontent.com/microsoft/dev-proxy/main/schemas/v0.21.0/rc.schema.json)) and therefore not being suggested with auto-completion. I guess we overread the property in `devproxy -h`, but at least it is listed there.

Apologies for the oversight! We'll fix it right away.

> Suggestion to maybe add it to the Getting Started section, since otherwise it is also pretty hidden in the Documentation :)

Let us have a look at how we could best fit it there. Thank you for the suggestions ❤

**Comment by waldekmastykarz** [comment]
I created two separate issues to track your suggestions. I'll close this one.

**Comment by waldekmastykarz** [comment]
@coinzz we've just updated the schema and the changes should be visible immediately. Once again, thank you for pointing it out and sorry for the trouble.
